src/Features/Core/Portable/StackTraceExplorer/DotnetStackFrameParser.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.CodeAnalysis;
+using Microsoft.CodeAnalysis.Text;
+
+namespace Microsoft.CodeAnalysis.StackTraceExplorer
+{
+    internal sealed class DotnetStackFrameParser : IStackFrameParser
+    {
+        private const string StackTraceAtStart = "at ";
```

Comment:
what happens if the user has a stack trace printed in a foreign language?

Reply:
I've been thinking about this for a while. We'll need to add a resource for this I think, but I want to make sure it copies the runtime resource. I don't think their string for this is exposed in a way we can easily access...